app/actions/fileLoader.js

```diff
@@ -165,6 +165,25 @@ export function playFile(file) {
   };
 }
 
+export function playFileFromFrame(filePath, startFrame) {
```

Comment:
This entire function shouldn't be needed anymore. You should update the `playFile()` action with the new interface and remove pretty much all references to `playFileFromFrame()`.

Reply:
but `playFile()` requires a file, not a string to a file path. How do you suggest I go about either getting a `file` object from the file path or making the function support file paths?